charts/vela-core/README.md

```diff
@@ -132,29 +132,39 @@ helm install --create-namespace -n vela-system kubevela kubevela/vela-core --wai
+| `logDebug`                                      | Enable debug logs for development purpose                                                                                                                          | `false`              |
+| `logFilePath`                                   | If non-empty, write log files in this path                                                                                                                         | `""`                 |
+| `logFileMaxSize`                                | Defines the maximum size a log file can grow to. Unit is megabytes. If the value is 0, the maximum file size is unlimited.                                         | `1024`               |
+| `kubeClient.qps`                                | The qps for reconcile clients                                                                                                                                      | `400`                |
+| `kubeClient.burst`                              | The burst for reconcile clients                                                                                                                                    | `600`                |
+| `authentication.enabled`                        | Enable authentication for application                                                                                                                              | `false`              |
+| `authentication.withUser`                       | Application authentication will impersonate as the request User                                                                                                    | `true`               |
+| `authentication.defaultUser`                    | Application authentication will impersonate as the User if no user provided in Application                                                                         | `kubevela:vela-core` |
+| `authentication.groupPattern`                   | Application authentication will impersonate as the request Group that matches the pattern                                                                          | `kubevela:*`         |
+| `sharding.enabled`                              | When sharding enabled, the controller will run as master mode. Refer to https://github.com/kubevela/kubevela/blob/master/design/vela-core/sharding.md for details. | `false`              |
+| `sharding.schedulableShards`                    | The shards available for scheduling. If empty, dynamic discovery will be used.                                                                                     | `""`                 |
+| `controller.readinessProbe.failureThreshold`    | Minimum consecutive failures for the [probe] to be considered failed after having succeeded                                                                        | `3`                  |
```

Comment:
Usually users do not need to configure these probe parameters. We can hide it from the README and only add it in the values.yaml.